zebra-network/src/peer/client.rs

```diff
@@ -75,7 +80,6 @@ impl Service<Request> for Client {
                     oneshot_recv_result
                         .expect("ClientRequest oneshot sender must not be dropped before send")
                 })
-                .instrument(self.span.clone())
```

Comment:
Not much benefit instrumenting this future, since it just waits for the oneshot to get a message. Instead the span is passed into the connection state machine.